### PR TITLE
Ruby 2.2 support. Drops support for Ruby 1.8.7.

### PR DESCRIPTION
**Pull request by bryantraywick**

### CODE REVIEW

File: spec/moonshine_generator_spec.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+eequire 'spec_helper'
```
Comment (uhlenbrock) [inline spec/moonshine_generator_spec.rb]: @bryantraywick, is that change intentional?

Comment (bryantraywick) [inline spec/moonshine_generator_spec.rb]: @uhlenbrock No, thanks it is fixed now. This should be rolling out later today, just need to fix Facter 2 compatibility in the popular moonshine plugins.
